server/routes/users.js

```diff
@@ -12,10 +7,5 @@ const router = express.Router();
 //   return res.send({ user: 'users' });
 // });
```

Comment:
unused code

Reply:
i believe there is "sample" written above that code. It was written there(by Grace) to guide people on how the route should look.